test/SConscript

```diff
@@ -0,0 +1,64 @@
+import os
+
+Import( 'env' )
+Import( '*' )
+
+if 'loudnessAnalyserLibStatic' in locals() and \
+    'loudnessCorrectorLibStatic' in locals() and \
```

Comment:
yes but you can also remove from dependencies. Tests will no covers correction of incorrect files.